packages/medusa/src/services/sales-channel-location.ts

```diff
@@ -90,7 +90,7 @@ class SalesChannelLocationService extends TransactionBaseService {
    * @param {string} salesChannelId - The ID of the sales channel.
    * @returns {Promise<string[]>} A promise that resolves with an array of location IDs.
    */
-  async listLocations(salesChannelId: string): Promise<string[]> {
+  async listLocationIds(salesChannelId: string): Promise<string[]> {
```

Comment:
suggestion: Line 97 add in the config `select: ["id"]` as we don't need the rest of the columns